plan/optimizer.go

```diff
@@ -113,6 +113,43 @@ func checkPrivilege(pm privilege.Manager, vs []visitInfo) bool {
 	return true
 }
 
+func setParents4FinalPlan(plan PhysicalPlan) {
```

Comment:
It's better not to put it in this file. This is only for explain. We can rename `explain_info.go` to `explain.go` and remove this funtion to that file.